lib/hca/enrollment_eligibility/parsed_statuses.rb

```diff
@@ -20,6 +20,7 @@ module ParsedStatuses
       INELIG_OVER65 = 'inelig_over65'
       INELIG_REFUSEDCOPAY = 'inelig_refusedcopay'
       INELIG_TRAINING_ONLY = 'inelig_training_only'
+      INELIG_OTHER = 'inelig_other'
```

Comment:
Small nit.  These are alphabetized.  Can you alphabetize this, and the new one in `ELIGIBLE_STATUSES`.